app/src/main/java/com/framgia/englishconversation/data/source/remote/timeline/TimelineRemoteDataSource.java

```diff
@@ -219,10 +253,14 @@ public void removeListener() {
     }
 
     public List<TimelineModel> getTimelineData(DataSnapshot dataSnapshot,
-            TimelineModel lastTimeline, UserModel userModel) {
+                                               TimelineModel lastTimeline, UserModel userModel) {
         List<TimelineModel> timelineModels = new ArrayList<>();
         for (DataSnapshot snapshot : dataSnapshot.getChildren()) {
             TimelineModel timelineModel = snapshot.getValue(TimelineModel.class);
+            if (timelineModel.getStatusModel() != null &&
+                    timelineModel.getStatusModel().getStatus() == Status.DELETE) {
+                    continue;
```

Comment:
ERROR: 'if' child have incorrect indentation level 20, expected level should be 16.